client/tests/print_resource_test.py

```diff
@@ -0,0 +1,66 @@
+import pytest
+import featureform as ff
+from featureform.resources import TrainingSet, Feature, ResourceColumnMapping
+from featureform.serving import ServingClient
+
+from .test_outputs import print_outputs
+
+from datetime import timedelta
+
+def test_feature_print():
+    feature = print_outputs["feature"]["unit_test"]["resource"]
```

Comment:
rename "resource" to "object" or "resource_object". Its unclear what this is fetching

Reply:
You may want to use fixtures to make it more clear
```
@pytest.fixture
def feature_object():
      return print_outputs["feature"]["unit_test"]["object"]

@pytest.fixture
def feature_expected_print():
      return print_outputs["feature"]["unit_test"]["output"]

def test_feature_print(feature_object, feature_expected_print):
       assert feature_expected_print == f"{feature_object}"
```

Reply:
changed to fixture schema